lib/std/os/windows/kernel32.zig

```diff
@@ -78,6 +78,26 @@ pub extern "kernel32" fn CreateFileW(
     hTemplateFile: ?HANDLE,
 ) callconv(WINAPI) HANDLE;
 
+pub extern "kernel32" fn CreateFileMappingA(
```

Comment:
https://github.com/ziglang/zig/issues/534

Reply:
I'll make that change.

As for the `Nt*` APIs, I'll need to do more research. A brief glance makes me think the appropriate tool is [NtCreateSection](https://undocumented.ntinternals.net/index.html?page=UserMode%2FUndocumented%20Functions%2FNT%20Objects%2FSection%2FNtExtendSection.html) and [NtMapViewOfSection](https://undocumented.ntinternals.net/index.html?page=UserMode%2FUndocumented%20Functions%2FNT%20Objects%2FSection%2FNtExtendSection.html). Presumably, this can be a follow-up PR.

Reply:
I ended up using the `Nt*` APIs to sidestep the issue I was having with wine not finding the `_UmapViewOfFile` symbol while linking.

Now, instead of exposing `CreateFileMapping` and `MapViewOfFile` in `std.os.windows`, it exposes `CreateSection` and  `MapViewOfSection`. These are Zig wrappers over the respective `Nt*` APIs.

*Resolving. I don't know if the general pattern here is for the reviewer or reviewee to resolve. Please let me know if this is a faux pas.*